pkg/ccl/backupccl/backup_test.go

```diff
@@ -11204,3 +11204,44 @@ func TestRestoreMemoryMonitoringMinWorkerMemory(t *testing.T) {
 	actualFingerprints := sqlDB.QueryStr(t, "SHOW EXPERIMENTAL_FINGERPRINTS FROM TABLE restore.bank")
 	require.Equal(t, expectedFingerprints, actualFingerprints)
 }
+
+// TestBoundBackupExportOnRanges tests that export requests never get sent
+// across range boundaries, even if the ranges are empty.
```

Comment:
I would maybe add something like: "We make this assertion by counting the number of export requests responses we've received."